flask_multipass/providers/ldap/providers.py

```diff
@@ -162,6 +162,11 @@ def __init__(self, *args, **kwargs):
         self._attributes = list(
             convert_app_data(self.settings['mapping'], {}, self.settings['identity_info_keys']).values())
         self._attributes.append(self.ldap_settings['uid'])
+        accepted_users = self.settings.setdefault('accepted_users', 'local').lower()
+        if accepted_users == 'all':
+            self.id_from_auth = True
+        else:
+            self.id_from_auth = False
```

Comment:
Why not make this a boolean setting?

    self.id_from_auth = self.settings.setdefault('auth_data_if_unknown'. False)

or something like that

Reply:
I don't think I have a good reason... when I did it a while ago I think I was not clear if other values may make sense and decided to go with this string value but revisiting this, I agree that a boolean would be easier. Your proposed name is may be a bit obscure but not easy to find a clear short name... What about `auth_data_if_identifier_missing`? longer but a bit more clear may be...

Reply:
Implemented in last version of the commit...